src/css/variables/colors.css

```diff
@@ -33,3 +33,6 @@ $alertColor__error: $color__angel;
 
 $spinnerColor__light: $color__river;
 $spinnerColor__dark: $color__white;
+
+$choiceGroupBackgroundColor: $color__white;
+$choiceGroupRadioButtonCheck: $color__river;
```

Comment:
We probably don't want to add more CSS variables. We want something more like this `theme.ts` file in Fabric: https://github.com/OfficeDev/office-ui-fabric-react/blob/660800656c40963e73450fddafb9d621ce089f4a/packages/styling/src/styles/theme.ts#L129